examples/src/main/java/org/bitcoinj/examples/FetchBlock.java

```diff
@@ -17,34 +17,71 @@
 
 package org.bitcoinj.examples;
 
+import joptsimple.OptionException;
+import joptsimple.OptionParser;
+import joptsimple.OptionSet;
 import org.bitcoinj.core.*;
+import org.bitcoinj.net.discovery.DnsDiscovery;
 import org.bitcoinj.params.TestNet3Params;
 import org.bitcoinj.store.BlockStore;
 import org.bitcoinj.store.MemoryBlockStore;
 import org.bitcoinj.utils.BriefLogFormatter;
 
 import java.net.InetAddress;
+import java.util.List;
 import java.util.concurrent.Future;
 
 /**
- * Downloads the block given a block hash from the localhost node and prints it out.
+ * Downloads the block given a block hash from the remote/localhost node and prints it out.
```

Comment:
Paragraphs are missing closing tags. Also, the first paragraph needs markup too.